source/_patterns/01-molecules/blocks/inline-sidebar.json

```diff
@@ -0,0 +1,7 @@
+{
+  "inline_sidebar_block_default": {
+    "title_accent": "Nulla Pariatur",
+    "title": "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris.",
+    "copy": "Pulvinar etiam non quam lacus suspendisse faucibus interdum. Vitae purus faucibus ornare suspendisse. Ligula ullamcorper malesuada proin libero nunc consequat interdum varius. Accumsan tortor posuere ac ut consequat semper viverra. Fermentum odio eu feugiat pretium. Et ligula ullamcorper malesuada proin libero nunc consequat. Venenatis urna cursus eget nunc scelerisque viverra mauris in. Volutpat sed cras ornare arcu dui. Quis eleifend quam adipiscing vitae proin sagittis nisl rhoncus. At consectetur lorem donec massa sapien faucibus et molestie ac. Maecenas accumsan lacus vel facilisis volutpat est velit. Nisl nisi scelerisque eu ultrices vitae auctor eu augue ut. Malesuada fames ac turpis egestas integer. Amet mattis vulputate enim nulla. Natoque penatibus et magnis dis. Ac orci phasellus egestas tellus rutrum tellus pellentesque eu tincidunt. Odio facilisis mauris sit amet massa vitae. Urna et pharetra pharetra massa massa. Nisl vel pretium lectus quam id leo in vitae turpis. Vestibulum lorem sed risus ultricies tristique nulla aliquet."
```

Comment:
Can you make this example work with more then one paragraph of text?

Reply:
@designerbrent - absolutely. Will get this back to you shortly.

Reply:
@designerbrent - I'll get this to you today. Still catching up from being out.

Reply:
@designerbrent - got the change in for multiple paragraphs. Let me know if this looks good.